library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -51,31 +52,35 @@ public EnvoyConfiguration(String statsDomain, int connectTimeoutSeconds, int dns
     this.appVersion = appVersion;
     this.appId = appId;
     this.virtualClusters = virtualClusters;
-    this.nativeFilters = nativeFilters;
+    this.nativeFilterChain = nativeFilterChain;
   }
 
   /**
    * Resolves the provided configuration template using properties on this
    * configuration.
    *
    * @param templateYAML the template configuration to resolve.
+   * @param platformFilterTemplateYAML helper template to build platform http filters.
+   * @param nativeFilterTemplateYAML helper template to build native http filters.
```

Comment:
these are not intended to be publicly replaced. I am guessing that @goaway added the platform string (and I copied him here) because the jni needs to be called to get the string. Vs. in objc you'll notice these are not params because objc can use the c string.

Reply:
Yep, that's right.